pybamm/input/parameters/lithium_ion/negative_electrodes/graphite_ORegan2021/graphite_LGM50_electrolyte_exchange_current_density_ORegan2021.py

```diff
@@ -0,0 +1,45 @@
+from pybamm import exp, constants, Parameter
+
+
+def graphite_LGM50_electrolyte_exchange_current_density_ORegan2021(c_e, c_s_surf, T):
+    """
+    Exchange-current density for Butler-Volmer reactions between graphite and LiPF6 in
+    EC:DMC.
+
+    References
+    ----------
+    .. [1] Kieran O’Regan, Ferran Brosa Planella, W. Dhammika Widanage, and Emma
+    Kendrick. "Thermal-electrochemical parametrisation of a lithium-ion battery:
+    mapping Li concentration and temperature dependencies." Journal of the
+    Electrochemical Society, submitted (2021).
+
+    Parameters
+    ----------
+    c_e : :class:`pybamm.Symbol`
+        Electrolyte concentration [mol.m-3]
+    c_s_surf : :class:`pybamm.Symbol`
+        Particle concentration [mol.m-3]
+    T : :class:`pybamm.Symbol`
+        Temperature [K]
+
+    Returns
+    -------
+    :class:`pybamm.Symbol`
+        Exchange-current density [A.m-2]
+    """
+
+    i_ref = 2.668  # (A/m2)
+    alpha = 0.792
```

Comment:
Is this consistent with the fact we only have symmetric Butler-Volmer?

Reply:
Probably not, from the thermodynamical point of view, but given that this was the measured exchange current density I think it is better to leave it as it is.